sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/JavaTypeInference.scala

```diff
@@ -123,7 +123,11 @@ object JavaTypeInference {
         val beanInfo = Introspector.getBeanInfo(typeToken.getRawType)
         val properties = beanInfo.getPropertyDescriptors.filterNot(_.getName == "class")
```

Comment:
Ah, sure. Then, let me ignore a property without the getter in `JavaTypeInference.inferDataType`, and allow empty property in `JavaTypeInference.serializerFor`/`deserializerFor`.